src/ApplePayGateway.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+
+class ApplePayGateway extends AbstractVindiciaGateway
```

Comment:
could you add a classblock here explaining how to use this gateways, like the other gateways have?